### DISCUSSION TITLE
Welcome to ZAWA Discussions!

### DISCUSSION THREAD

**Discussion by Mnesikos** [ZAWA-Evolved/ZAWA#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together. 💪

  Please keep in mind this is not the place to report bugs or issues with the mod! Please utilize the [Issues ](https://github.com/ZAWA-Evolved/ZAWA/issues)feature instead.

  We also have a [Discord](https://discord.gg/HnSMawZ5Rq) if you wish to actively partake in our community!

**Comment by SophieMOA** [comment]
How would i go about changing the config so that enrichment doesnt go down, we like the hunger aspect and all our animals are eating and drinking, they just arent getting enrichment?
